README.md

```diff
@@ -210,12 +223,12 @@ BuildKit can also be used by running the `buildkitd` daemon inside a Docker cont
 To run daemon in a container:
 
 ```
-docker run -d --privileged -p 1234:1234 tonistiigi/buildkit --addr tcp://0.0.0.0:1234
+docker run -d --privileged -p 1234:1234 moby/buildkit --addr tcp://0.0.0.0:1234
```

Comment:
We could probably raise the priority of this block now that we have actual release images.

Reply:
Lets explain the master branch as well.